interface/frontend/src/components/open-image/OpenDICOM.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="container">
+    <div class="col-xs-6">
+      <div class="DICOM-container">
+        <div class="DICOM" ref="DICOM"></div>
+      </div>
+      <p>{{ dicom.imageId}}</p>>
+    </div>
+  </div>
+</template>
+
+<script>
+  var cornerstone = require('cornerstone-core')
+  var Q = require('q')
+
+  export default {
+    name: 'open-dicom',
+    data () {
+      return {
+        dicom: {
+          imageId: '',
+          minPixelValue: 0,
+          maxPixelValue: 255,
+          slope: 1.0,
+          intercept: 0,
+          windowCenter: 90,
+          windowWidth: 100,
+          render: cornerstone.renderGrayscaleImage,
+          getPixelData: this.getPixelData,
+          rows: 512,
+          columns: 512,
+          height: 512,
+          width: 512,
+          color: false,
+          base64data: '',
+          columnPixelSpacing: 1,
+          rowPixelSpacing: 1,
+          sizeInBytes: 512 * 512 * 2
+        },
+        dicomUrl: 'LIDC://LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192/-95.000000.dcm',
+        info: null,
+        csName: 'LIDC'
+      }
+    },
+    watch: {
+      info: function (val) {
+        if (val != null) {
+          this.applyMeta(val)
+          console.log(this.dicom)
+          this.loadImage(this.resolveDICOM)
+        }
+      }
+    },
+    mounted: function () {
+      this.fetchData(this.dicomUrl)
+    },
+    methods: {
+      fetchData (id) {
+        this.$axios.get('/api/images/metadata?dicom_location=/images/' + id.slice(this.csName.length + 3))
```

Comment:
Can we not let Django generate the URIs instead?

Reply:
Sorry, not sure that I understand it correctly: did you mean that this URI prefix should be computed by the Django and then provided to vue?